tests/external_processing/test_asm_blocking.py

```diff
@@ -0,0 +1,97 @@
+from utils import scenarios, features, irrelevant
+from tests.appsec import test_request_blocking as base_request_blocking
+from tests.appsec import test_blocking_addresses as base_blocking_addresses
+
+
+@features.not_reported
+@scenarios.external_processing_blocking
+class Test_ExternalProcessing_ASM_AppSecRequestBlocking(base_request_blocking.Test_AppSecRequestBlocking):
+    pass
+
+
+@features.not_reported
+@scenarios.external_processing_blocking
+class Test_ExternalProcessing_ASM_Blocking_client_ip(base_blocking_addresses.Test_Blocking_client_ip):
+    pass
+
+
+@features.not_reported
+@scenarios.external_processing_blocking
+class Test_ExternalProcessing_ASM_Blocking_request_method(base_blocking_addresses.Test_Blocking_request_method):
+    @irrelevant(True, reason="The endpoint /tag_value is not implemented in the weblog")
```

Comment:
Then it's a missing_feature: 

```suggestion
    @missing_feature(True, reason="The endpoint /tag_value is not implemented in the weblog")
```

Reply:
Applied in 1eb5853 😄